### PR TITLE
Fix error description in variable.js

### PR DESCRIPTION
**Pull request by Bouh**
fix error description at line 94 to 100.
Before the description in the same as 77 to 83

### CODE REVIEW

File: GDJS/Runtime/variable.js [hunk 1]
```diff
@@ -92,11 +92,11 @@ gdjs.Variable.prototype.getChild = function(childName) {
...
 /**
- * Get the child with the specified name.
+ * Return the child in a variable.
  *
- * If the variable has not the specified child, an empty variable with the specified name
- * is added as child.
- * @method getChild
+ * If the variable has the specified children
+ * @method hasChild
+ * @return true if variable has a children
```
Comment (4ian) [inline GDJS/Runtime/variable.js]: the children with the specified name :)

File: GDJS/Runtime/variable.js [hunk 2]
```diff
@@ -92,11 +92,11 @@ gdjs.Variable.prototype.getChild = function(childName) {
 };
 
 /**
- * Get the child with the specified name.
+ * Return the child in a variable.
  *
- * If the variable has not the specified child, an empty variable with the specified name
- * is added as child.
- * @method getChild
+ * If the variable has the specified children
```
Comment (4ian) [inline GDJS/Runtime/variable.js]: Only the last sentence should be preserved:
`Check if the variable has the specified children`.

**Comment by 4ian** [comment]
Hey @Bouh, nice catch and thanks for the PR :D 
I think we can still improve a bit the description ;) I've made some comments, you can update the merge and I'll merge it after 👍

**Comment by Bouh** [comment]
Est-ce bon ?
Je comprend pas tout de github encore.

**Comment by 4ian** [comment]
Oui c'est bon :) Je "squash et merge" Thanks!

**Pull request merged**